src/plotman/archive.py

```diff
@@ -130,13 +140,41 @@ def archive(dir_cfg, all_jobs):
         (archdir, freespace) = sorted(available)[index]
 
     if not archdir:
-        return(False, 'No archive directories found with enough free space')
-    
+        return 'No archive directories found with enough free space'
+
+    arch_jobs = get_running_archive_jobs(dir_cfg['archive'])
```

Comment:
Can you pull from `development` on this branch? There have been changes to how we reference configs internally, so references to `dir_cfg["archive"]` should all use dot notation now: `dir_cfg.archive`